guides/doc-Converting_a_Host_to_RHEL/master.adoc

```diff
@@ -2,7 +2,7 @@ include::common/attributes.adoc[]
 include::common/header.adoc[]
 :context: convert-to-rhel
 
-= Converting a Host to RHEL
+= {ConvertinHostRHELDocTitle}
```

Comment:
```suggestion
= {ConvertingHostRHELDocTitle}
```

Please also check the original attribute for the missing `g`.

Reply:
Deffo! Thanks!